test/e2e/operator_test.go

```diff
@@ -51,8 +51,7 @@ func TestDefaultClusterDNSExists(t *testing.T) {
 			APIVersion: "dns.openshift.io/v1alpha1",
 		},
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      "default",
-			Namespace: "openshift-cluster-dns-operator",
+			Name: "openshift-dns",
```

Comment:
This change is in the "Rename operator and dns namespaces" commit but should be in the "Make DNS CRD cluster scoped" commit.